### PR TITLE
Cleanbot fixes and adjustments

### PR DESCRIPTION
**Pull request by MTandi**
## About The Pull Request

Fixed cleanbots not prioritizing adjacent tiles.
Made them able to clean 1 tile away (they have that arm, just like medibots), enabling them cleaning under the tables.
Expanded the list of cleanable objects for cleanbots with missing dirt/trash types.

Comparison with 20x speed:

https://user-images.githubusercontent.com/3625094/200519843-29ffd814-7e9a-4a0f-ba2c-8c434ce959f8.mp4

## Why It's Good For The Game

Cleanbots had non-working code that made them clean very inefficiently, running around, instead of going 1 by 1 tile next to them. Now they do a bit better job at this and also don't leave the mess under the tables/machinery.

## Changelog
:cl:
fix: Cleanbots properly prioritize adjacent trash
qol: Cleanbots now clean floor under the tables
qol: Cleanbots now clean more types of trash
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/cleanbot.dm [hunk 1]
```diff
@@ -285,29 +285,27 @@
...
 				return
+
+		if(target && path.len == 0 && (get_dist(src,target) > 1))
+			path = get_path_to(src, target, 30,id=access_card)
 			mode = BOT_MOVING
-		else if(!bot_move(target))
-			target = null
-			mode = BOT_IDLE
+			if(!path.len) //try to get closer if you can't reach the target directly
+				path = get_path_to(src, target, 30,1,id=access_card)
```
Comment (Mothblocks) [inline code/modules/mob/living/simple_animal/bot/cleanbot.dm:299]: Named parameter for 1 (and probably 30 as well)

Comment (MTandi) [reply code/modules/mob/living/simple_animal/bot/cleanbot.dm:299]: Why? Do you want me to change this for all bots or just cleanbots?

Comment (Mothblocks) [reply code/modules/mob/living/simple_animal/bot/cleanbot.dm:299]: https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#prefer-named-arguments-when-the-meaning-is-not-obvious

Feel free to only change your own, but cleaning up the others would be good

**Review by Mothblocks — APPROVED** [review]

**Comment by calliophis-maculiceps** [comment]
JOHNNN JOHNNN WE CANT MERGE PRS UNTIL 515, OH NO HE HAS HEADPHONES ON

**Pull request merged**